tests/KdybyTests/nette-reset.neon

```diff
@@ -13,6 +13,10 @@ common:
 		cacheStorage:
 			class: Nette\Caching\Storages\MemoryStorage
 
+		-
+			class: KdybyTests\ValidatorMock\CustomConstraintValidator
+			tags:
+				kdyby.validator.constraintValidator: KdybyTests\ValidatorMock\CustomConstraintValidator
```

Comment:
With some additional magic `- kdyby.validator.constraintValidator` would be enough. Thoughts?